scripts/export_producers_platform_data_to_public_database.sh

```diff
@@ -4,28 +4,51 @@
 
 export PERL5LIB="lib:${PERL5LIB}"
 
-PRODUCERS=(
-org-barilla-france-sa
-org-ferrero-france-commerciale
-org-unilever-france-gms
-org-unilever-france-rhd
-org-nestle-france
-org-panzani-sa
-org-cristalco
-org-materne
-org-garofalo-france
-org-brasseries-kronenbourg
-org-carrefour
-org-lustucru-frais
-org-nestle-waters
-org-kambly
-org-kambly-france
-org-saint-hubert
-org-d-aucy
-org-lea-nature
-org-auchan-apaw
-org-les-mousquetaires
-)
+filter_organizations_that_have_automated_export() {
```

Comment:
Shan't we leverage having orgs in MongoDB now and just make a mongoDB request (you can even user mongo client) to get all orgs with activate_automated_daily_export_to_public_platform on ?

cc: @stephanegigandet

Reply:
Oh yes I forgot, sure we can do that.